get_smarter_api_clients/geag.py

```diff
@@ -2,9 +2,25 @@
 Client for GetSmarter API Gateway.
 """
 
+from get_smarter_api_clients.oauth import OAuthApiClient
```

Comment:
nit: should this be `getsmarter_api_clients.oauth` (and in the directory name)?

Reply:
yeah getsmarter is one word 😵‍💫